source/index.md

```diff
@@ -11,11 +13,57 @@ build Ember apps, focusing on the most widely-used features of Ember.js.
 For exhaustive documentation of every Ember feature and API, see the
 [Ember.js API documentation](http://emberjs.com/api/).
 
-These guides are written in Markdown and are
-[available on GitHub](https://github.com/emberjs/guides/).
-If there is something missing, or you find a typo or
-mistake, please help us by filing an issue or submitting a pull
-request. You can easily jump to Github's editor for any page by
-clicking the pencil icon in the upper right corner.
+## Assumptions
+
+While we try to make the guides as beginner-friendly as we can, we must
+establish a baseline so that the guides can keep focused on Ember.js
+functionality. We will try to link to appropriate documentation whenever
+a concept is introduced.
+
+To make the most out of the guides, you should have a working knowledge of:
+
+* **HTML, CSS, JavaScript** - the building blocks of web pages. You can find documentation of each of these technologies at the [Mozilla Developer Network][mdn]
+* **Promises** - the native way to deal with asynchrony in your JavaScript code. [TDK: need a good link][promises]
+* **ES2015 modules** - you will better understand Ember CLI's project structure and import paths if you are comfortable with [ES6 JavaScript Modules][js-modules]
+* **ES2015 syntax** - Ember CLI comes with Babel.js by default so you can
+take advantage of newer language features such as arrow functions, template
+strings, destructuring, and more. You can check the
+[Babel.js documentation][babeljs] or read [Understanding ECMAScript 6][es6]
+online.
+
+## Reporting a problem
+
+Typos, missing words, code samples with errors are all considered
+documentation bugs. If you spot one of them, or want to otherwise improve
+the existing guides, we are happy to help you help us!
+
+Some of the more common ways to report a problem with the guides are:
+
+* Using the pencil icon on the top-right of each guide page
+* Opening an issue/pull request to [the GitHub repository][gh-guides]
+
+Clicking the the pencil icon will bring you to GitHub's editor for that
+guide so you can edit right away, using the Markdown markup language.
+This is the fastest way to correct a typo, a missing word, or an error in
+a code sample.
+
+If you wish to make a more significant contribution be sure to check our
+[issue tracker][gh-guides-issues] to see if your issue is already being
+addressed. If you don't find an active issue, open a new one.
+
+If you have any styling questions, or about the contributing process you
+can check out our [contributing guide][gh-guides-contributing]. If your
+question persists, reach us at `#documentation` on the Slack
+group.
 
 Good luck!
+
+[mdn]: https://developer.mozilla.org/en-US/docs/Web
+[promises]: TDK
```

Comment:
Neither am I. And it's a 1,6 MB download :-1: 
I would consider it if it were a standalone page with just the Promises stuff.